LandSandBoat/server
# [JAs] Update functionality for WAR JA Blood Rage

**_I affirm:_**

## What does this pull request do?



- Creates new mod "ENHANCES_BLOOD_RAGE" in status.lua and modifier.h to provide Blood Rage duration bonus from gear.
- Updates documentation/mods_by_id.txt with new mod and new SPARE.
- Adds "ENHANCES_BLOOD_RAGE" mod to duration calc in job_utils/warrior.lua.
- Adds Job Point "BLOOD_RAGE_EFFECT" to power calc in job_utils/warrior.lua.
- Updates Blood Rage effect onEffectGain and onEffectLose functions with getPower.
- Adds "ENHANCES_BLOOD_RAGE" mod to Ravager's Lorica +1/+2 and Boii Lorica NQ/+1 to item_mods.sql.
- Sets the Warcry status effect to be overwritten by the Blood Rage status effect, and vice versa, in status_effects.sql.

**References**
https://www.bg-wiki.com/ffxi/Blood_Rage

## Steps to test these changes



1. Apply the changes to your server.
2. Add `player:PrintToPlayer(string.format("Crit Hit Rate: %i ", target:getMod(xi.mod.CRITHITRATE)), xi.msg.channel.SYSTEM_3)` to commands/getstats.lua.
3. Log in and change to WAR using `!changejob WAR`.
4. Use `!additem 11084`, `!additem 11184`, `!additem 26898`, and `!additem 26899` to give yourself the WAR gear.
5. Give your character Job Points using `!setjobpoints ###`, then max the "Blood Rage Effect" category.
6. Equip the gear to increase Blood Rage duration and use `!getstats <me>` to check your Crit Hit Rate improvements.
